node/network/protocol/src/request_response/mod.rs

```diff
@@ -102,12 +109,30 @@ const POV_REQUEST_TIMEOUT_CONNECTED: Duration = Duration::from_millis(1200);
 /// fit statement distribution within a block of 6 seconds.)
 const STATEMENTS_TIMEOUT: Duration = Duration::from_secs(1);
 
+/// We want to time out backed candidate requests to time out relatively fast,
+/// because slow requests will bottleneck the backing system. Ideally, we'd have
```

Comment:
I haven't really gotten around to the request/response parts of this protocol, but yes, we will probably not be requesting the same candidate from multiple peers at the same time. At least in the first draft. This is important, especially for large statements, to save bandwidth.

Reply:
Oh yeah, I am definitely not suggesting here to fetch the same candidate from multiple peers, but rather for different candidates. With regards to throughput, if one validator is slow on fetching one candidate it should not matter much if it is able to download others faster because of that under-utilization. Latency should also matter less with async backing.